Valet.podspec

```diff
@@ -11,19 +11,8 @@ Pod::Spec.new do |s|
   s.frameworks = 'Security'
   s.ios.deployment_target = '9.0'
   s.tvos.deployment_target = '9.0'
+  s.watchos.deployment_target = '2.0'
   s.macos.deployment_target = '10.11'
 
-  s.test_spec 'Tests' do |test_spec|
```

Comment:
@dnkoutso do you know how to get the test spec to ignore `watchos`? Since the watch doesn't have the XCTest framework (or really any test infra), we need to skip it. I tried including `test_spec.watchos.exclude_files = 'Tests/**/*.{h,m,swift}'` below [in a previous commit](https://github.com/square/Valet/pull/127/commits/ed0355a2a9efad28fd3fc7041c43a3174b707f48), but I still end up with the following error: `/Users/travis/Library/Developer/Xcode/DerivedData/App-azgcswumbhaldvdurgqjunerahug/Build/Products/Debug-watchsimulator/Valet-Unit-Tests.xctest: bundle format unrecognized, invalid, or unsuitable`

You can see the full error here: https://api.travis-ci.org/v3/job/348993626/log.txt

Reply:
@dnkoutso could I get around this by adding a dummy test file for the watchOS target?

Reply:
Nope. And just saw https://github.com/CocoaPods/CocoaPods/issues/7009